### PR TITLE
Shift templates

### PR DESCRIPTION
**Pull request by spaaacetoast**
Add shift templates with a calender overview

### CODE REVIEW

**Comment by spaaacetoast** [comment]
Its a big one but shift template works, this also adds (fixes) the posibility to delete shifts when they contain sections and adds a calender overview

File: packages/payload/package.json [hunk 1]
```diff
@@ -10,24 +10,30 @@
     "build": "next build",
     "lint": "next lint",
     "payload": "payload",
-    "start": "next start"
+    "start": "next start",
+    "generate:importmap": "PAYLOAD_CONFIG_PATH=payload.config.ts payload generate:importmap",
+    "prepare": "panda codegen"
```
Comment (baer95) [inline packages/payload/package.json:15]: This line fails during the docker build because `panda` is not available with production dependencies

Comment (baer95) [reply packages/payload/package.json:15]: `prepare` is automatically executed by bun after `install`. Can we call this script something else, maybe `generate:panda`?

Comment (spaaacetoast) [reply packages/payload/package.json:15]: panda codegen needs to be run before building yeah, ill make sure that panda is installed for the prod build then!

Comment (baer95) [reply packages/payload/package.json:15]: the thing is we do an `install --production` to have a smaller build output, so we will have to do a multi stage dockerfile that installs once with all dependencies for the build, but then does another install with prod dependencies only for the final image

File: Dockerfile.astro [hunk 2]
```diff
@@ -4,7 +4,6 @@ WORKDIR /home/bun/app
 
 COPY package.json bun.lockb ./
 COPY packages/astro/package.json ./packages/astro/package.json
-COPY packages/payload/package.json ./packages/payload/package.json
```
Comment (baer95) [inline Dockerfile.astro:7]: I know this is the astro dockerfile, but the `install` command is run on root level so it requires all sub-packages. We can remove this only if bun can install a single workspace.

**Review by baer95 — APPROVED** [review]

**Review by baer95 — APPROVED** [review]

**Pull request merged**